### ISSUE TITLE
See if incrementing the action.yml can be automated

### ISSUE THREAD

**Issue by marcofranssen** [philips-labs/slsa-provenance-action#66]
When releasing a new version of our action we push a new tag.

This results in a release of the docker image with that specific tag as well a github release.

Now to ensure people are using this tag the exact same commit referenced by the tag should also point to the to-be released version of our image within action.yml

https://github.com/philips-labs/slsa-provenance-action/blob/github-releases/action.yaml#L29

Basically we have the chicken and egg issue that needs to be resolved to be able to automate this.

Currently we will have to make a new commit on the main branch increasing the version, then tag this commit and do

```shell-session
git push --follow-tags
```

**Comment by marcofranssen** [comment]
Resolved by #85 #87
